src/main/java/com/epam/lowcost/controller/UserController.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.lowcost.controller;
+
+import com.epam.lowcost.model.User;
+import com.epam.lowcost.service.UserServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+@Controller()
+@RequestMapping(value = "/user")
+
+public class UserController {
+
+    @Autowired
+    UserServiceImpl userService;
+
+    @GetMapping(value = "/all")
+    public String listAllUsers(Model model)
+    {
+        model.addAttribute("users",userService.getAllUsers());
+        return "users";
+    }
+    @PostMapping (value = "/id")
+    public String getById(@RequestParam long id, Model model)
```

Comment:
HTTP POST method with getting ? I think you you did it by accident, you should be more careful) Before click "commit" please check all your code: style, logic and other